docs/guides/migration_mechanism.md

```diff
@@ -9,32 +9,32 @@ keywords:
 - Firefly
 - wallet
 - legacy nodes
+- explanation
 ---
 
 # How the IOTA Chrysalis Phase 2 Token Migration Works
 
-For the transition to Chrysalis Phase 2, the IOTA Foundation had to develop a way for funds to be migrated to the new
-network. The reason a migration mechanism was necessary was because the previous network no longer supported WOTS addresses.
+For the Chrysalis Phase 2 transition, the IOTA Foundation had to develop a way for funds to be migrated to the new network. Because the previous network did not support the WOTS addresses, this new mechanism was created.
```

Comment:
```suggestion
For the Chrysalis Phase 2 transition, the IOTA Foundation developed a way to migrate funds to the new network. IOTA created this new mechanism because the previous network did not support the WOTS addresses.
```